src/services/profile/profile-router.ts

```diff
@@ -272,7 +268,7 @@ profileRouter.put("/points", strongJwtVerification, async (req: Request, res: Re
 
     await AttendeeProfileModel.updateOne({ userId: decodedData.id }, update);
 
-    return res.status(Constants.SUCCESS).send({ points: profile.points });
+    return res.status(Constants.SUCCESS).send({ userId: decodedData.id, points: profile.points });
```

Comment:
make sure these are adding (+= points, should accept positive or negative) not setting (= points)